pkgs/tools/misc/qmk/default.nix

```diff
@@ -27,13 +40,23 @@ buildPythonApplication rec {
     appdirs
     argcomplete
     colorama
-    dotty-dict
+    #dotty-dict # See https://pypi.org/project/qmk-dotty-dict/
+    qmk-dotty-dict
     hid
     hjson
     jsonschema
     milc
     pygments
+    pyrsistent
     pyusb
+  ] ++ [ # Binaries need to be in the path so this is in propagatedBuildInputs
```

Comment:
Please wrap instead.

Reply:
There's already a wrapper in `buildPythonApplication` that adds things to the `PATH`, it doesn't make sense to me to wrap the same binary twice